libnd4j/include/cblas.h

```diff
@@ -32,7 +32,7 @@
 #define CBLAS_H
 #endif
 
-#ifdef HAVE_MKLDNN
+#ifdef HAVE_ONEDNN
 // FIXME: latest mkldnn doesn't ship mklml anymore?
 // include CBLAS from MKL-DNN
 //#include <mkl_cblas.h>
```

Comment:
Got an answer to that question?

Reply:
@treo MKLMKL was deprecated long ago. Look at the latest publish: https://anaconda.org/anaconda/mklml MKLDNN became onednn and is now what's shipped with oneapi.

Reply:
@treo removed the reference here. Feel free to check the source tree where the mkldnn compat layer is bundled: https://github.com/KonduitAI/oneDNN/blob/9cd49946cddb6936d12963d94eb1dbb2170ff3ac/include/mkldnn.h